bin/wasm-node/javascript/demo/demo.cjs

```diff
@@ -0,0 +1,179 @@
+// Smoldot
```

Comment:
We don't care about the demo being CommonJS or ES modules. I don't want to duplicate 180 lines of code just because one uses `require` and the other `import`.